src/Features/Core/Portable/SignatureHelp/SignatureHelpItem.cs

```diff
@@ -20,29 +20,30 @@ internal class SignatureHelpItem
         /// </summary>
         public bool IsVariadic { get; }
 
-        public ImmutableArray<SymbolDisplayPart> PrefixDisplayParts { get; }
-        public ImmutableArray<SymbolDisplayPart> SuffixDisplayParts { get; }
+        public ImmutableArray<TaggedText> PrefixDisplayParts { get; }
+        public ImmutableArray<TaggedText> SuffixDisplayParts { get; }
 
         // TODO: This probably won't be sufficient for VB query signature help.  It has
         // arbitrary separators between parameters.
-        public ImmutableArray<SymbolDisplayPart> SeparatorDisplayParts { get; }
+        public ImmutableArray<TaggedText> SeparatorDisplayParts { get; }
 
         public ImmutableArray<SignatureHelpParameter> Parameters { get; }
 
-        public ImmutableArray<SymbolDisplayPart> DescriptionParts { get; internal set; }
+        public ImmutableArray<TaggedText> DescriptionParts { get; internal set; }
 
-        public Func<CancellationToken, IEnumerable<SymbolDisplayPart>> DocumentationFactory { get; }
+        public Func<CancellationToken, IEnumerable<TaggedText>> DocumentationFactory { get; }
 
-        private static readonly Func<CancellationToken, IEnumerable<SymbolDisplayPart>> s_emptyDocumentationFactory = _ => SpecializedCollections.EmptyEnumerable<SymbolDisplayPart>();
+        private static readonly Func<CancellationToken, IEnumerable<TaggedText>> s_emptyDocumentationFactory =
+            _ => SpecializedCollections.EmptyEnumerable<TaggedText>();
 
         public SignatureHelpItem(
```

Comment:
Can you keep both constructors to make the migration for TS a little less jarring...